docs/notebooks/Word2Vec_Multistream.ipynb

```diff
@@ -0,0 +1,212 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Word2Vec Multistream API Tutorial\n",
+    "\n",
+    "This tutorial introduces new **`corpus_file`** argument for **`gensim.models.word2vec.Word2Vec`** model and how to use it. \n",
+    "\n",
+    "## Motivation\n",
+    "\n",
+    "Because standard Word2Vec training with `sentences` argument doesn't scale so well when the number of workers is large, special `corpus_file` argument was added to tackle this problem. Training with `corpus_file` yields **significant performance boost** (training is 300% faster with 14 workers in comparison to training with `sentences` argument). Also, it outruns performance of original Word2Vec C tool in terms of words/sec processing speed.\n",
+    "\n",
+    "While providing such benefits in performance, `corpus_file` argument accepts path to your corpus file which must be in a format of `gensim.models.word2vec.LineSentence` (one sentence per line, words are separated by whitespaces).\n",
+    "\n",
+    "\n",
+    "**Note**: you have to build `gensim` with Cython optimizations (`FAST_VERSION >= 0`) in order to be able to use `corpus_file` argument."
```

Comment:
Probably it's a good idea to add a new variable `MULTISTREAM_VERSION` for your implementation (in case if the user can compile standart `_inner`, but can't compile your version, how exactly happened with CI)

Reply:
👍